network/forks/genesis/pubsub_cfg.go

```diff
@@ -1,8 +1,10 @@
 package genesis
 
 import (
+	"encoding/binary"
+
 	"github.com/bloxapp/ssv/network/forks"
-	scrypto "github.com/bloxapp/ssv/utils/crypto"
```

Comment:
I think maybe we can delete it this package or use it and move the xxhash implementation to it